packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook-jobs.job.ts

```diff
@@ -42,6 +42,9 @@ export class CallWebhookJobsJob {
 
   @Process(CallWebhookJobsJob.name)
   async handle(data: CallWebhookJobsJobData): Promise<void> {
+    // If you change that function, double check it does not break Zapier
```

Comment:
ok for now, I think we should share types between server and zapier (likely a package twenty-shared with ts interfaces later)

Reply:
Would love such a package